pkgs/development/libraries/fribidi/default.nix

```diff
@@ -1,19 +1,23 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig }:
+{ stdenv
+, fetchurl
+
+, meson
+, ninja
+, pkgconfig
+}:
 
 stdenv.mkDerivation rec {
-  name = "fribidi-${version}";
-  version = "0.19.7";
+  name = "${pname}-${version}";
+  pname = "fribidi";
+  version = "1.0.4";
 
-  src = fetchFromGitHub {
-    owner = "fribidi";
-    repo = "fribidi";
-    rev = version;
-    sha256 = "10q5jfch5qzrj2w4fbkr086ank66plx8hp7ra9a01irj80pbk96d";
+  # NOTE: 2018-06-06 v1.0.4: Only URL tarball has "Have pre-generated man pages: true", which works-around upstream usage of rare ancient `c2man` fossil.
+  src = fetchurl {
+    url = "https://github.com/${pname}/${pname}/releases/download/v{$version}/${name}.tar.bz2";
```

Comment:
This looks wrong

Reply:
The URL or the comment? ;]

Ok, all further changes I going to do on the next day.

Reply:
The `{$version}` should be `${version}`.